public/src/components/channelManagement/testEditorContextTargeting.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```

Comment:
Most of this is copied from epicTestTargetContentEditor.tsx.
It now uses the `PageContextTargeting` model